react-components/src/communities/view/EditCommunity.js

```diff
@@ -469,16 +469,11 @@ class EditCommunity extends Component {
                         })}
                         messages={messages.messages}
                         message={
-                            <div>
-                                {loading && (
-                                    <CircularProgress
-                                        size={30}
-                                        classes={{ root: classes.loading }}
-                                        thickness={7}
-                                    />
-                                )}
-                                {getMessage("confirmDeleteCommunity")}
-                            </div>
+                            <LoadingMask loading={loading}>
+                                <div>
+                                    {getMessage("confirmDeleteCommunity")}
+                                </div>
```

Comment:
Minor, but are these added `<div>` tags needed?

Reply:
I actually moved the `div` tag that was outside the `CircularProgress` into the `LoadingMask`. But ya I can check to see if this `div` can be removed.